src/compass/defaults/s1_cslc_geo.yaml

```diff
@@ -82,9 +82,9 @@ runconfig:
               # Secondary number of iterations
               extraiter: 10
               # Enable/disable computation of latitude raster
-              compute_latitude: True
+              compute_latitude: False
               # Enable/disable computation of longitude raster
-              compute_longitude: True
+              compute_longitude: False
               # Enable/disable computation of height raster
               compute_height: True
```

Comment:
For some reason, I had the height disabled and it was still appearing in my static layers

Reply:
> For some reason, I had the height disabled and it was still appearing in my static layers

@seongsujeong did you encounter the same error as @vbrancat?  I was unable to replicate the error.

@vbrancat were you testing with the latest from this branch?